src/main/java/core/basesyntax/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReaderService {
```

Comment:
The class `FileReaderImpl` should not start with an empty line. According to the checklist, remove the empty line at the beginning of the class implementation.